SONAR-9374 Update page titles for SonarCloud mode

gregaubert opened the pull request.

stas-vilchik commented on server/sonar-web/src/main/js/app/components/DefaultHelmetContainer.js:
```diff
@@ -18,13 +18,21 @@
   return (
     <div>
-      <Helmet defaultTitle="SonarQube" />
+      <Helmet defaultTitle={isSonarCloud ? 'SonarCloud' : 'SonarQube'} />
```
`isSonarCloud` can be `null` or `{ value: 'true' }` or `{ value: 'false' }`. When it's `{ value: 'false' }`, the title is wrong.

gregaubert replied:
:scream: Thanks!

gregaubert commented:
@stas-vilchik I fixed it as well as the `GlobalFooter`, and also renamed `isSonarCloud` back to `sonarqubeDotCom` so we will not miss it when doing the global renaming ticket.
